examples/java/src/main/java/org/apache/beam/examples/cookbook/IcebergRestCatalogCDCExample.java

```diff
@@ -201,5 +226,17 @@ public interface IcebergCdcOptions extends GcpOptions {
     String getCatalogName();
 
     void setCatalogName(String catalogName);
+
+    @Description("Trigger the streaming write example")
+    @Default.Boolean(false)
+    boolean getTriggerStreamingWrite();
+
+    void setTriggerStreamingWrite(boolean triggerStreamingWrite);
+
+    @Description("The Pub/Sub topic to read from for the streaming write example")
+    @Default.String("projects/pubsub-public-data/topics/taxirides-realtime")
+    String getTopic();
+
+    void setTopic(String topic);
```

Comment:
nit: This should probably be a static final variable (in `IcebergRestCatalogStreamingWriteExample`), since the operations we're doing here rely on the data shape coming from that topic. I think no need to make it a pipeline option